src/main/main.c

```diff
@@ -311,8 +311,14 @@ void init(void)
     spiInit(SPIDEV_1);
 #endif
 #ifdef USE_SPI_DEVICE_2
+#ifdef OMNIBUSF4
```

Comment:
@blckmn , I think it was worth giving hardware revisions a shot. Also by trying something you see what the limitations are.